clang/include/clang/AST/Expr.h

```diff
@@ -3860,6 +3860,7 @@ class CStyleCastExpr final
 class BinaryOperator : public Expr {
   enum { LHS, RHS, END_EXPR };
   Stmt *SubExprs[END_EXPR];
+  bool ExcludedOverflowPattern = false;
```

Comment:
I'd prefer to avoid increasing the size of BinaryOperator if possible; do we really need to precompute this?  If we do, is there a spare bit in subclass data?

Reply:
Ok, I think I found a spare bit in `BinaryOperatorBits` but I am not super familiar with how all these bit classes work.

See: https://github.com/llvm/llvm-project/pull/100272/commits/cb0fcd1c3b1ded7258bb20cff4d187ca56f17149 

Thanks